pages/about.html

```diff
@@ -4,9 +4,53 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Awesome Sunny</title>
+  <link href="../styles/style.css" rel="stylesheet">
 </head>
 <body>
-  
+  <header> 
+    <h1 class="page-titles">About Sunny</h1>
+    <nav>
+      <ul class="container">
+        <li>
+          <a href = "portfolio.html">Portfolio</a> 
+        </li>
+        <li>
+          <a href = "index.html"> Index</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <p class="about">
+    Sunny is a coder who ponders much<br>
+    about the intersection of technology, art,<br>
+    ethics, and her role in a gloablized world.<br>
+    Sunny received her bachelors in Women and<br>
+    Gender studies from San Francisco State<br>
+    University in 2019. Recieving her bachelors<br>
+    awakened Sunny to the endless paths in which<br>
+    we may create a more equitable world. Sunny<br>
+    belives creating inclusive accessible technology<br>
+    is one such path. Sunny is skilled in Python, SQL,<br>
+    Flask, HTML, CSS, Javascript and<br>
+    is always learning more!</p>
+    <p class="about-2">
```

Comment:
The use of `about-2` here to distinguish this about section would make more sense if this was an ID instead of a class. IDs need to be unique per page, but classes can be repeated. There are a few other places where I think you can put IDs where you used classes as they are not repeated in the page.

The distinction to me comes that if you will need to repeat the same style on multiple elements, a class makes the most sense, otherwise, if this is identifying a single uniquely identifiable element, then an ID is the best fit. It's definitely wavy, and there's a lot of HTML and CSS out there that does not maintain this distinction.